Sources/AccessibilitySnapshot/Core/Swift/Classes/UIAccessibility+SnapshotAdditions.swift

```diff
@@ -128,6 +128,20 @@ extension NSObject {
             traitSpecifiers.append(strings.tabTraitName)
         }
 
+        if accessibilityTraits.contains(.textEntry) {
+            if accessibilityTraits.contains(.scrollable) {
+                // This is being a UITextView
+                if let accessibilityValue = accessibilityValue, !accessibilityValue.isEmpty {
+                    // No-op. iOS does not read the trait name when there's text.
```

Comment:
That what iOS does, I found the experience pretty weird 🤷

Reply:
Hmm, on my iOS 13 test device it's still reading the trait. I'll test on my 12 and 14 as well, just need to charge them up. 😬